FlowCal/gate.py

```diff
@@ -134,14 +134,14 @@ def high_low(data, channels=None, high=None, low=None, full_output=False):
 
     # Default values for high and low
     if high is None:
-        if hasattr(data_ch, 'domain'):
-            high = [np.Inf if di is None else di[-1] for di in data_ch.domain()]
+        if hasattr(data_ch, 'range'):
+            high = [np.Inf if di is None else di[1] for di in data_ch.range()]
             high = np.array(high)
         else:
             high = np.Inf
     if low is None:
-        if hasattr(data_ch, 'domain'):
-            low = [-np.Inf if di is None else di[0] for di in data_ch.domain()]
+        if hasattr(data_ch, 'range'):
+            low = [-np.Inf if di is None else di[0] for di in data_ch.range()]
```

Comment:
is there a description somewhere that actually specifies that `range[0]` is the low and `range[1]` is the high? kinda obvious, but might be nice to have that explicitly stated somewhere